connector_importer_api/models/sources/import_source_api.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2025 Binhex <https://www.binhex.cloud>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from ast import literal_eval
+from itertools import chain
+
+from odoo import Command, api, fields, models
+
+
+class ImportSourceApi(models.Model):
+    _name = "import.source.api"
+    _inherit = ["import.source", "connector.importer.api.mixin"]
+    _description = "API import source"
+    _source_type = "api"
+
+    company_id = fields.Many2one(
+        "res.company", required=True, default=lambda self: self.env.company
+    )
+    enviroment = fields.Selection(
+        selection=[("test", "Test"), ("prod", "Production")],
+        default="test",
+        required=True,
+    )
+    enviroment_url = fields.Char(string="Enviroment URL", required=True)
+    type_request = fields.Selection(
+        string="Request type",
+        selection=[("get", "GET")],
```

Comment:
Why not include the POST method as one of the options for the type_request field?
```suggestion
        selection=[("get", "GET"), ("post", "POST")],
```

Reply:
Hi @adasatorres , thanks for your comments. Initially, the POST type was not added because it was not necessary for the initial requirement, but it can still be extended and all possible types can be added and the necessary configurations made.